contracts/Api3ToUsdReader.sol

```diff
@@ -0,0 +1,19 @@
+//SPDX-License-Identifier: MIT
```

Comment:
The ClaimsManager manager can replace this with something like
```
contract Api3ToUsdReader is IApi3ToUsdReader {
    function read() external view override returns (int224) {
        return 10**18; // Fixes API3/USD to 1
    }
}
```

Reply:
I see, so the idea here is to have logic that can be swapped, wrapped in a contract function. So `ClaimsManager` will always call the same `IApi3ToUsdReader.read()` interface in `convertUsdToApi3`.